tests/dummy/app/helpers/i.js

```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
+
+const matcher = /\$\{\s*(.*?)\s*\}/g;
+
+export function interpolate([ str ], hash) {
```

Comment:
why did you copy and paste the code from the interpolate helper versus just using the helper in a template?